.github/workflows/docker_linux.yml

```diff
@@ -0,0 +1,54 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: Docker-Linux
+
+on:
```

Comment:
Can we tadd a condition to trigger the yml only when docker/linux/Dockerfile get modified.

Reply:
Just kicked off a build with this condition, should be good, but a little hard to test until the first action is merged into this repo